File: PinYoGo_parent/pinyogo_interface/src/main/java/cn/itcast/core/service/seckill/SeckillGoodsService.java
package cn.itcast.core.service.seckill;

import cn.itcast.core.pojo.seckill.SeckillGoods;

import java.util.List;

public interface SeckillGoodsService {

    List<SeckillGoods> findSeckillGoodsList();

    SeckillGoods findOneFromRedis(Long id);
}

File: PinYoGo_parent/pinyogo_web_seckill/src/main/webapp/js/service/seckillGoodsService.js
app.service('seckillGoodsService',function($http){

    this.findSeckillGoodsList = function() {
        return $http.get("seckillGoods/findSeckillGoodsList.do");
    };

    this.findOne = function(id) {

        return $http.get("../seckillGoods/findOne.do?id="+id);

    };
});